chef/cookbooks/nova/recipes/compute.rb

```diff
@@ -154,7 +154,8 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
 
       if node[:nova][:libvirt_type] == "kvm"
         if node.platform_version.to_f >= 12.0
-          package "qemu-kvm"
+          package "qemu-kvm" if node[:architecture] == "x86_64"
+          package "qemu-ppc" if node[:architecture] =~ "ppc64le"
```

Comment:
@vuntz I was wrong. qemu-kvm is just a compatibility package providing just a wrapper around qemu-system-$ARCH as qemu-kvm 

It most probably not needed even on x86